src/blog/introducing-truffle-dashboard/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Introducing Truffle Dashboard - Stop copy + pasting your private keys!
+hide:
+  - navigation
+---
+
+![truffle dashboard - banner](./truffle-dashboard-private-keys.png)
+
+**By [Kingsley Arinze](https://twitter.com/heydamali)**
+
```

Comment:
Do we need an "Introduction" H2 here? The ToC on the right looks a bit weird without it.

Reply:
Agreed